src/sentry/web/api.py

```diff
@@ -91,6 +91,160 @@ def wrapped(request, *args, **kwargs):
     return wrapped
 
 
+def process_event(event_manager, project, key, remote_addr, helper, attachments):
```

Comment:
Everything follows comes (unchanged) from `StoreView.process` except that the signal sender changed to `sender=process_event`. I see no non-test connected signals so nobody should care about the sender changing?

Reply:
Why do we send the signal if there are no receivers?

Reply:
Good question, it looks like use them in `getsentry` (and maybe other users do too). But I don't see anyone depending on the sender.